photon-client/src/types/SettingTypes.ts

```diff
@@ -138,6 +138,9 @@ export interface CameraCalibrationResult {
   distCoeffs: JsonMatOfDouble;
   observations: BoardObservation[];
```

Comment:
Does the backend now send an empty array or nothing for observations? If it is nothing, this needs to be marked as optional or removed.

Reply:
empty and non null